e2e/playwright/setup-environment

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+cd ../../../
+git clone --depth 1 https://github.com/mattermost/mattermost.git
+cd mattermost
```

Comment:
Note to other reviewers that this script is only to help set up the development environment, and is not used for CI. See this directory's README for more info